### ISSUE TITLE
Naming of 2-functoriality

### ISSUE THREAD

**Issue by mikeshulman** [HoTT/HoTT#1143]
The action of a functor `X_functor` on homotopies is variously called `X_functor_homotopy`, `X_functor_htpy`, and `X_2functor`.  We should be consistent.

**Comment by Alizter** [comment]
I think `X_2functor` is fine. I would start with changing the O_functor's in ReflectiveSubuniverse to a more streamlined convention first as this seems to cover the shape of just about every functoriality condition we use in the library. For example O_functor_2homotopy can be made O_3functor etc.

I might write a guide for which functoriality conditions you should expect, i.e. how to write the naturality and pseudnatuIrality conditions. I don't know if there are any real differences between "lax n-functors" and "pseudonatural n-functors" apart from various things from the former are inverted at various levels in the latter. I doubt I will be able to get past n=2 however.

**Comment by mikeshulman** [comment]
Yes, there's no difference between lax and pseudo functors for (oo,1)-categories where we are, since there are no noninvertible 2-cells.
